cmd/manager/main.go

```diff
@@ -247,6 +189,75 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "AtlasDatabaseUser")
 		os.Exit(1)
 	}
+
+	cfg := mgr.GetConfig()
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		setupLog.Error(err, "unable to create clientset")
+		os.Exit(1)
+	}
+
+	dbaasInstalled, err := dbaas.CheckDBaaSCRDInstalled(clientset)
```

Comment:
can we move this section in a separate file like here done in [crunchy operator](https://github.com/CrunchyData/crunchy-bridge-operator/blob/master/dbaas_support.go)

Reply:
Good idea! Moved it to a separate function.